packages/@vue/cli-plugin-e2e-webdriverio/index.js

```diff
@@ -0,0 +1,71 @@
+const path = require('path')
+
+const { info, chalk, execa } = require('@vue/cli-shared-utils')
+const { cmdArgs } = require('@wdio/cli/build/commands/run')
+
+const CLI_OPTIONS = Object.entries(cmdArgs).reduce((obj, [param, { desc }]) => {
+  obj[`--${param}`] = desc
+  return obj
+}, {})
+
+module.exports = (api, options) => {
+  api.registerCommand('test:e2e', {
+    description: 'run end-to-end tests with WebdriverIO',
+    usage: 'vue-cli-service test:e2e [options]',
+    options: {
+      '--remote': 'Run tests remotely on SauceLabs',
+      '--headless': 'Run tests headless',
+      ...CLI_OPTIONS
+    },
+    details:
+      `All WebdriverIO CLI options are also supported.\n` +
+      chalk.yellow(`https://webdriver.io/docs/clioptions.html`)
+  }, (args, rawArgs) => {
+    return Promise.all([
+      startDevServer(args, api)
+    ]).then((results) => {
+      const { server, url } = results[0]
+      info(`Running end-to-end tests...`)
+
+      // expose dev server url to tests
+      if (!args.baseUrl) {
+        rawArgs.push(`--baseUrl=${url}`)
+      }
+
+      const configFile = !args.remote
+        ? path.join(process.cwd(), 'tests', 'wdio.local.conf.js')
+        : path.join(process.cwd(), 'tests', 'wdio.sauce.conf.js')
```

Comment:
The `tests` part seems extraneous? As the config files are at the project root according to the templates.